Blish HUD/Controls/TextInputBase.cs

```diff
@@ -64,7 +64,10 @@ public abstract class TextInputBase : Control {
         /// </summary>
         public event EventHandler<ValueEventArgs<bool>> InputFocusChanged;
 
-        protected void OnTextChanged(ValueChangedEventArgs<string> e) => TextChanged?.Invoke(this, e);
+        protected void OnTextChanged(ValueChangedEventArgs<string> e) {
+            OnPropertyChanged(nameof(Text));
```

Comment:
This PR may not be appropriate for this change, but in any case, this is already called in `SetText` when `SetProperty` is called.  The issue is just that the callername is `SetText` instead of `Text`.  The fix is to update `if (!SetProperty(ref _text, value)) return false;` to include the explicit property name instead of it pulling it with CallerMemberName.